.github/workflows/release.yml

```diff
@@ -21,15 +21,16 @@ jobs:
         uses: pypa/cibuildwheel@v2.16.5
         env:
           CIBW_SKIP: "cp36-* cp37-* pp* *musl*"
-          CIBW_TEST_SKIP: "*-macosx_arm64 *-macosx_universal2:arm64 *-win32 *-manylinux_i686 cp310-manylinux*"
+          CIBW_TEST_SKIP: "*-macosx_arm64 *-macosx_universal2:arm64 *-win32 *-win_amd64 *-manylinux_i686 cp310-manylinux*"
```

Comment:
I'm fine with this in the short term so we can get a functional 2.6.x release out with wheels. But longer term we should get to the bottom of the windows build failures.